.gitignore

```diff
@@ -0,0 +1,6 @@
+.DS_Store
```

Comment:
I'd prefer not to - nvm isn't used by all node users, and npm-debug.log only occurs on an install error. The no-local-gitignore approach has worked very well in practice, since there are few drawbacks to a global gitignore.